python/cog/server/runner.py

```diff
@@ -42,12 +42,13 @@ def setup(self):
         Sets up the predictor in a subprocess. To start a prediction after
         setup call `run()`.
         """
-        self.predictor_process = multiprocessing.Process(
+        self.predictor_process = multiprocessing.get_context("spawn").Process(
```

Comment:
Same here, would be nice to document `multiprocessing.get_context("spawn").Process? It's a fairly unusual pattern and we might forget the rationale.

Also, why is it `"spawn"` here and `"fork"` in the base class of `LogProcess`?

Reply:
Comment added about `get_context` and about the choice of methods used.